api/src/services/sheets/sheets.js

```diff
@@ -0,0 +1,56 @@
+import { requireAuth } from 'src/lib/auth'
+import {
+  updateRestaurant,
+  restaurantByOwnerId,
+} from '../restaurants/restaurants'
+
+import { google } from 'googleapis'
+import fs from 'fs'
+
+export const createSheet = async () => {
+  requireAuth()
+
+  const auth = new google.auth.GoogleAuth({
+    credentials: {
+      client_email: process.env.CLIENT_EMAIL,
+      private_key: process.env.PRIVATE_KEY.replace(/\\n/gm, '\n'),
+    },
+    scopes: ['https://www.googleapis.com/auth/drive'],
+  })
+
+  const authClient = await auth.getClient()
+
+  const driveClient = google.drive({
+    version: 'v3',
+    auth: authClient,
+  })
+  const sheet = await driveClient.files.create({
+    resource: {
+      name: 'SheetMenu.csv',
+      mimeType: 'application/vnd.google-apps.spreadsheet',
+    },
+    media: {
+      mimeType: 'text/csv',
+      body: fs.createReadStream('src/templates/sheetmenu-template.csv'),
+    },
+  })
+
```

Comment:
Can we add a check that we have a sheet id? If we don't, we shouldn't continue below.